packages/oid4vci-holder/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sphereon/ssi-sdk.oid4vci-holder",
-  "version": "0.30.1",
+  "version": "0.30.99-bram",
```

Comment:
Where is the PR for this version? It needs to be reviewed & merged first so this can be updated.

Reply:
local testing, reverted